addons/gdUnit3/src/update/GdUnitUpdate.gd

```diff
@@ -172,6 +178,20 @@ func _on_update_pressed():
 	queue_free()
 	plugin.queue_free()
 
+static func _extract_package(source :String, dest:String) -> Result:
+	var err = OS.execute("tar", ["-xf%s" % source, "-C%s" % dest])
```

Comment:
```suggestion
	var err = OS.execute("tar", ["-xf", source, "-C", dest])
```
As far as I can see, the way it is now would not have spaces between `-xf` and the source path, and `-C` and the destination path.

Reply:
yes using separat argument is the right way. this was just a try ;)